lib/childrensize.js

```diff
@@ -10,13 +10,22 @@ var rectStore = null;
 function toArray (nodeList) {
     // Array.prototype.slice does not work on HTMLCollection in IE8
     var arr = [];
-    for (var i=0, l=nodeList.length; i<l; i++) { arr.push(nodeList[i]); }
+    if (nodeList) {
```

Comment:
I would prefer `if (!nodeList) { return []; }` to avoid unnecessary indentation.